src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery.stack;
+
+public class IntArrayStack implements Stackable {
+
+    private static final int MAX_STACK_SIZE = 12;
+
+    private int[] numbers = new int[MAX_STACK_SIZE];
+
+    private int total = EMPTY_STACK_VALUE;
+
+    @Override
+    public void push(final int input) {
+        if (!isFull())
+            numbers[total++] = input;
```

Comment:
Changing EMPTY_STACK_VALUE value requires such fixes? It does not make sense. Incorrect fix.